plugin/storage/cassandra/factory.go

```diff
@@ -131,6 +137,262 @@ func (f *Factory) configureFromOptions(o *Options) {
 	}
 }
 
+const (
+	MODE               = `MODE`
+	DATACENTER         = `DATACENTER`
+	TRACE_TTL          = `TRACE_TTL`
+	DEPENDENCIES_TTL   = `DEPENDENCIES_TTL`
+	KEYSPACE           = `KEYSPACE`
+	REPLICATION_FACTOR = `REPLICATION_FACTOR`
+	VERSION            = `VERSION`
+	COMPACTION_WINDOW  = `COMPACTION_WINDOW`
+)
+
+// Parameters required for initilizing the db
+type StorageConfigParams struct {
+	mode                   string
+	datacenter             string
+	trace_ttl              int
+	dependencies_ttl       int
+	keyspace               string
```

Comment:
this should already be defined in the main config

Reply:
Are you talking about this config: jaeger/cmd/jaeger/config-cassandra.yaml ? How would I know what all config key-value pairs are already defined?

Reply:
Hi @yurishkuro , I needed some help here. Is this: jaeger/cmd/jaeger/internal/all-in-one.yaml the example for the config file which would be used in v2? If yes, under what section should I expect the cassandra specific configs to be put? I couldn't find any example for this in the current code base.

Just to get the complete picture, would the schema configs I am adding look something like this:
schema:
    datacenter: <datacenterName>
    keyspace: <keyspace>
    .......

Reply:
cmd/jaeger/config-cassandra.yaml

Reply:
Let me know if the current place of defining it works?